internal/services/servicebus/servicebus_queue_data_source.go

```diff
@@ -137,6 +115,20 @@ func dataSourceServiceBusQueue() *pluginsdk.Resource {
 			},
 		},
 	}
+
+	if !features.FivePointOh() {
+		for _, name := range []string{"batched_operations", "express", "partitioning"} {
```

Comment:
I'm pretty split on this block of code. I can see the reason since it's mostly copy paste but the readability is rough but it's going away soon anyway so what's the harm. 

For consistency and readability, I'd say we have the individual blocks but we can bring it up internally to discuss if we want to do anything like this going forward

Reply:
No worries, just another case of "Wyatt's Over-Engineering (TM)"